scripts/test.ts

```diff
@@ -25,6 +25,11 @@ require('source-map-support').install({
 });
 
 
+interface CoverageLocation {
```

Comment:
This should be part of the `test: add an istanbul bootstrap script` commit.

Reply:
It should. I was just pushing stuff as quickly as I could :( sorry. I normally clean up before submitting for review.

Reply:
I was leaving the comment here mostly for myself when reworking this PR. I'll move it to the other commit.